specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2018-12-20/examples/AzureIaasVm/GetAccessToken_Get.json

```diff
@@ -7,12 +7,16 @@
     "containerName": "IaasVMContainer;iaasvmcontainerv2;rshhtestmdvmrg;rshmdvmsmall",
     "protectedItemName": "VM;iaasvmcontainerv2;rshhtestmdvmrg;rshmdvmsmall",
     "recoveryPointId": "26083826328862",
-    "api-version": "2018-12-20-preview",
+    "api-version": "2018-12-20",
     "parameters": {
       "properties": {
         "tenantId": "33e01921-4d64-4f8c-a055-5bdaffd5e33d",
         "audience": "https://RecoveryServices/IaasCoord/aadmgmt/ase",
-        "servicePrincipalObjectId": "5ecd8123-cf74-4037-83e9-9246b227b351"
+        "servicePrincipalObjectId": "5ecd8123-cf74-4037-83e9-9246b227b351",
+        "identityDetails": {
+          "isSystemAssignedIdentity": false,
+          "managedIdentityArmUrl": "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/asmaskarRG1/providers/Microsoft.ManagedIdentity/userAssignedIdentities/asmaskartestmsi"
```

Comment:
> "managedIdentityArmUrl": "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/asmaskarRG1/providers/Microsoft.ManagedIdentity/userAssignedIdentities/asmaskartestmsi" [](http://example.com/codeflow?start=9&length=181)

You might want to rename this property to managedIdentityResourceId since the example you have is a resource Id and not a URL(with host and other information)

Reply:
We use it as part of URL by prefixing with ARM endpoint for all crud operations on the resource.

Reply:
How its used internally is not relevant.  All resource IDs can be turned into URLs by prepending a hostname.  The string is a resource ID, not a URL.  Other properties in this object are also resource IDs and use a "resourceId" suffix.  Either change this property name to "managedIdentityId" or "managedIdentityResourceId".

Reply:
done, please check